Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8
+FROM node:12.5.0-alpine
```

Comment:
i think we should move from node 8 since it's reaching it's LTS end

Reply:
:+1: